skimage/filters/rank/tests/test_rank.py

```diff
@@ -351,11 +401,26 @@ def test_compare_8bit_vs_16bit(self, method):
         image16 = image8.astype(np.uint16)
         assert_equal(image8, image16)
 
+        np.random.seed(0)
+        volume8 = np.random.randint(128, high=256,
+                                    size=(10, 10, 10), dtype=np.uint8)
+        volume16 = volume8.astype(np.uint16)
+        assert_equal(volume8, volume16)
```

Comment:
this assertion isn't necessary

Reply:
I'm not sure about the difference between the image assertion versus the volume one. @ryanlu41 ?

Reply:
@jni I removed the assertion but not sure why; it is the same assertion that 2d uses.